udata_youckan.py

```diff
@@ -106,7 +106,7 @@ def authorized():
     response = youckan.get('me')
     data = response.data
 
-    user = datastore.find_user(slug=data['slug'])  # TODO: use user id instead
+    user = datastore.find_user(email=data['email'])
```

Comment:
I think, it might requires another PR that stores the ID (for the flux) and a manual migration that use an extract from YouCKAN DB to feed the ID on the stock